src/main/java/spring/rest/task/controllers/EmailController.java

```diff
@@ -0,0 +1,45 @@
+package spring.rest.task.controllers;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.web.bind.annotation.*;
+import spring.rest.task.services.EmailService;
+import java.io.IOException;
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+
+@RestController(value = "/emails")
+public class EmailController {
+    EmailService emailService;
+
+    public EmailController(EmailService emailService) {
+        this.emailService = emailService;
+    }
+
+    @GetMapping
+    public List<SimpleMailMessage> getAllEmails(){
+       return emailService.getAllEmails();
+    }
+    @PostMapping(value = "emails/new")
```

Comment:
URL is a name. You will not call you dog new, right? The same goes to url: it should be a noun. Not a verb! Use request method to explain what to do. Example: POST /emails - means that you post(or add to simplify) something to collection of emails. Here is the good explanation of how to create good urls https://www.restapitutorial.com/lessons/restfulresourcenaming.html

Besides that you duplicate emails part which makes url even more awkward: /emails/emails/new